samtranslator/model/sam_resources.py

```diff
@@ -455,11 +456,7 @@ def _construct_role(self, managed_policy_map, event_invoke_policies):
         if self.Tracing:
             # use previous (old) policy name for regular regions
```

Comment:
You can move these two comments to the new `get_xray_managed_policy_name()` function

Reply:
done